backend/app.py

```diff
@@ -308,6 +315,178 @@ def create_machine():
     return jsonify({"message": "Success!"}), 200
 
 
+@app.route("/change_machine", methods=["POST"])
+def change_machine():
+    """
+    Receives a request for change of a machine and opens a PR
+    """
+    machine_id = int(request.args.get("id"))
+    title = str(request.args.get("title")).strip()
+    address = str(request.args.get("address")).strip()
+    area = str(request.args.get("area")).strip()
+    status = str(request.args.get("status")).strip()
+    latitude = float(request.args.get("lat_coord"))
+    longitude = float(request.args.get("lon_coord"))
+
+    # Load server locations and find existing machine info
+    server_locations, latest_commit_sha = load_latest_json()
+    (
+        existing_machine_infos,
+        index_in_server_locations,
+    ) = find_machine_in_database(machine_id, server_locations["features"])
+
+    # Start new dictionary
+    updated_machine_entry = existing_machine_infos.copy()
+    updated_machine_entry["properties"]["last_updated"] = str(datetime.today()).split(
+        " "
+    )[0]
+    change_message = "Changed"
+
+    # Case 1: status was changed:
+    if status != existing_machine_infos["properties"]["machine_status"]:
+        updated_machine_entry["properties"]["machine_status"] = status
+        change_message += " status,"
+
+    # Case 2: if area was changed -> match to available areas
+    if area != existing_machine_infos["properties"]["area"]:
+        # Identify area
+        area, score = fuzzysearch.extract(area, COUNTRIES, limit=1)[0]
+        if score < 90:
+            return (
+                jsonify(
+                    {
+                        "error": "Could not match country. Provide country or US state name in English"
+                    }
+                ),
+                400,
+            )
+        updated_machine_entry["properties"]["area"] = area
+        change_message += " area,"
+
+    # Case 3: Title changed
+    if title != existing_machine_infos["properties"]["name"]:
+        updated_machine_entry["properties"]["name"] = title
+        change_message += " title,"
+
+    # Case 4: multimachine changed
+    try:
+        multimachine = int(request.args.get("multimachine"))
+    except ValueError:
+        # just put the multimachine as a string, we need to correct it then
+        multimachine = "TODO" + str(request.args.get("multimachine"))
+    if multimachine != 1:
```

Comment:
```suggestion
    if multimachine != existing_machine_infos["properties"]["multimachine"]:
```

Reply:
this would yield a keyerror because the multimachine key is usually missing for existing machines. I'll implement a different solution

Reply:
then use `existing_machine_infos["properties"].get("multimachine", 1)

Reply:
that was the plan